nextcord/ui/select/channel.py

```diff
@@ -89,8 +93,9 @@ def __init__(
         min_values: int = 1,
         max_values: int = 1,
         disabled: bool = False,
-        row: Optional[int] = None,
+        defaults: Optional[List[Union[SelectDefault, GuildChannel, PartialMessageable]]] = None,
```

Comment:
We need to accept them as MentionableSelect's have 2 different types and in the decorator, you might not have a role object or whatsoever

Reply:
I'm fine with `MentionableSelect` accepting them (should have mentioned that initially), but to me it doesn't make sense for the others to accept them too. Maybe the others could accept an `int` or an `Object` as a static placeholder for the actual object, as you mentioned?

Reply:
I'm not sure with that, just for consistency reasons. Maybe we could do classmethods called `SelectDefault.user(1234)` for example, which automatically fills in the type? Because honestly it's hard to determine where int's or Object's are supported (User, Channel, Role) and where they aren't (Mentionable).

Reply:
That sounds like a good idea.

Reply:
resolved as I added that as a feature now